org.gnome.Eolie.json

```diff
@@ -67,333 +62,297 @@
+      }]
     },
     {
       "name": "packaging",
       "buildsystem": "simple",
       "build-commands": [
-        "pip3 install --prefix=/app packaging-16.8-py2.py3-none-any.whl"
+        "pip3 install --prefix=/app packaging-19.0-py2.py3-none-any.whl"
       ],
-      "sources": [
-        {
-          "type": "file",
-          "url": "https://pypi.python.org/packages/87/1b/c39b7c65b5612812b83d6cab7ef2885eac9f6beb0b7b8a7071a186aea3b1/packaging-16.8-py2.py3-none-any.whl#md5=c7326351bf015fa53c74b0075923ab02",
-          "sha256": "99276dc6e3a7851f32027a68f1095cd3f77c148091b092ea867a351811cfe388"
-        }
-      ]
+      "sources": [{
+        "type": "file",
+        "url": "https://files.pythonhosted.org/packages/91/32/58bc30e646e55eab8b21abf89e353f59c0cc02c417e42929f4a9546e1b1d/packaging-19.0-py2.py3-none-any.whl",
+        "sha256": "9e1cbf8c12b1f1ce0bb5344b8d7ecf66a6f8a6e91bcb0c84593ed6d3ab5c4ab3"
+      }]
     },
     {
       "name": "asn1crypto",
       "buildsystem": "simple",
       "build-commands": [
-        "pip3 install --prefix=/app asn1crypto-0.22.0-py2.py3-none-any.whl"
+        "pip3 install --prefix=/app asn1crypto-0.24.0-py2.py3-none-any.whl"
       ],
-      "sources": [
-        {
-          "type": "file",
-          "url": "https://pypi.python.org/packages/97/ba/7e8117d8efcee589f4d96dd2b2eb1d997f96d27d214cf2b7134ad8acf6ab/asn1crypto-0.22.0-py2.py3-none-any.whl#md5=5f8356d63fb715160b00fe764f6790ea",
-          "sha256": "d232509fefcfcdb9a331f37e9c9dc20441019ad927c7d2176cf18ed5da0ba097"
-        }
-      ]
+      "sources": [{
+        "type": "file",
+        "url": "https://files.pythonhosted.org/packages/ea/cd/35485615f45f30a510576f1a56d1e0a7ad7bd8ab5ed7cdc600ef7cd06222/asn1crypto-0.24.0-py2.py3-none-any.whl",
+        "sha256": "2f1adbb7546ed199e3c90ef23ec95c5cf3585bac7d11fb7eb562a3fe89c64e87"
+      }]
     },
     {
       "name": "WebOb",
       "buildsystem": "simple",
       "build-commands": [
-        "pip3 install --prefix=/app WebOb-1.6.0-py2.py3-none-any.whl"
+        "pip3 install --prefix=/app WebOb-1.8.5-py2.py3-none-any.whl"
       ],
-      "sources": [
-        {
-          "type": "file",
-          "url": "https://pypi.python.org/packages/12/5a/b6ef092cf8e70d2d63fdd8d2c6d479b1896d815cb256585213ed86e10196/WebOb-1.6.0-py2.py3-none-any.whl#md5=8ff68c5fce011abd8c3ef714a24450ec",
-          "sha256": "69356a75980172f3e76717f6899b6c75d95ccdc7a2cb6a3b8bd6a7de3fc72401"
-        }
-      ]
+      "sources": [{
+        "type": "file",
+        "url": "https://files.pythonhosted.org/packages/06/e1/4acd2b4327fceb4c6446bdbca515f807ab83188526fd654940c00bcf8cc3/WebOb-1.8.5-py2.py3-none-any.whl",
+        "sha256": "36db8203c67023d68c1b00208a7bf55e3b10de2aa317555740add29c619de12b"
+      }]
     },
     {
       "name": "idna",
       "buildsystem": "simple",
       "build-commands": [
-        "pip3 install --prefix=/app idna-2.5-py2.py3-none-any.whl"
```

Comment:
Is it not possible to separate the python dependencies and update it automatically using the flatpak script generator [script](https://github.com/flatpak/flatpak-builder-tools/blob/master/pip/flatpak-pip-generator) ?

Reply:
@biswasab Well, upstream needs to use a requirements.txt file for that, which is not the case right now :(

Reply:
@bilelmoussaoui I think if this PR get merged it will break PR #18 . Btw have you tested if firefox sync works with #18 or not?